### ISSUE TITLE
Enabling Accessibility TTS freezes the program because `espeak` is not accessible from the sandbox

### ISSUE THREAD

**Issue by geekley** [flathub/org.libretro.RetroArch#252]
RetroArch on Linux uses `espeak` to provide TTS (text-to-speech). Source: "How to enable accessibility" section in [this post](https://www.libretro.com/index.php/retroarch-1-8-2-coming-soon-accessibility-features-for-blind-people/).

But the program crashes because the flatpak doesn't include it, and the sandbox won't allow access to the host's espeak command-line. So, you could bundle [espeak](http://espeak.sourceforge.net/), or even simpler, just allow access to this command if it's installed on the host.

### Fix
I managed to make it work on my Kubuntu.
First, espeak should already be installed on the host (`sudo apt install espeak`).
Create executable text file at this flatpak's bin folder. For user installation, it's at:
`~/.local/share/flatpak/app/org.libretro.RetroArch/current/active/files/bin/espeak`:
```sh
#!/usr/bin/env bash
flatpak-spawn --host espeak "$@"

```
To allow it to run `flatpak-spawn --host`:
```sh
flatpak override --user --talk-name=org.freedesktop.Flatpak org.libretro.RetroArch
```
This workaround is just for testing, and will be overwritten on upgrades, I believe.
So, please include this fix to allow it to run `espeak` out-of-the-box.

I don't know if there's a better way to do it (e.g. some "accessibility" flatpak permission that does this).
If you don't want to add this permission, you could also bundle espeak somehow.

### Test
```sh
$ flatpak run --command=sh org.libretro.RetroArch
[📦 org.libretro.RetroArch ~]$ espeak "Hello, world!"
```
You should hear it. Also, go to RetroArch and toggle **Settings > Accessibility > Accessibility Enable > ON** then move in the menu. You should hear the narrator.
With this fix, I'm assuming that running "espeak" command is the only thing required by RetroArch on Linux, ~~but I haven't tested any other languages, just English~~. I did test a second language and confirmed that it works correctly, using the proper voice/pronunciation.

**Comment by RobLoach** [comment]
Good catch! I definitely think we should have `espeak` available.

In the mean time, this happens outside Flathub too, so I created this issue: https://github.com/libretro/RetroArch/issues/15328

**Comment by geekley** [comment]
@RobLoach I tested this again, and I'm pretty sure the upstream crash has been solved.
Can `espeak` be included now?
Should be simple enough to add at least the wrapper as explained above.
Or, if possible, bundle it somehow, to avoid the extra permission.
